tests/bin/get_att.c

```diff
@@ -0,0 +1,18 @@
+#include "libc.h"
+
+int main(void) {
+    // TODO: Good buffer length?
+    unsigned char nonce[512];
+    unsigned char buf[512];
+    size_t technology;
+
+    void* nonce_ptr = &nonce;
+    void* buf_ptr = &buf;
+    size_t* tech_ptr = &technology;
+
+    ssize_t rax = get_att(nonce_ptr, sizeof(nonce), buf_ptr, sizeof(buf), tech_ptr);
+
+    // TODO: Currently this should return ENOSYS. However, this will change
+    // as get_att() is implemented for each technology.
+    return rax != ENOSYS;
```

Comment:
`return rax != -ENOSYS;`

Notice the negative. The Linux syscall ABI for x86_64 specifies that the kernel returns values in `rax` and `rdx` unless an error occurs. Then it returns a negative errno in `rax`.

Reply:
This needs to be updated to check `errno`.